.travis.yml

```diff
@@ -161,9 +160,3 @@ script:
       cd $INSTALL_DIR/docs/iris/src/whatsnew;
       python aggregate_directory.py --checkonly;
     fi
-  - if [[ $TEST_TARGET == 'coding' ]]; then
-      cd $INSTALL_DIR;
-      python setup.py pyke_rules;
-      python setup.py std_names;
```

Comment:
Ohhh @pelson ... in hindsight, did we need the `python setup.py pyke_rules` and `python setup.py std_names` at line 120 above.... I'll check